includes/class-sensei-settings.php

```diff
@@ -460,8 +460,8 @@ public function init_fields() {
 		);
 
 		$fields['course_author'] = array(
-			'name'        => __( 'Display Course Author', 'sensei-lms' ),
-			'description' => __( 'Output the Course Author on Course archive and My Courses page.', 'sensei-lms' ),
+			'name'        => __( 'Display Course Author', 'sensei-lms' ) . ' ' . __( '(legacy pages)', 'sensei-lms' ),
+			'description' => __( 'Output the Course Author on Course archive and My Courses page.', 'sensei-lms' ) . ' ' . __( 'If you are using blocks for these pages, you need to remove the author from the blocks.', 'sensei-lms' ),
```

Comment:
Maybe I'm missing something, so just out of curiosity, is there any particular reason for concatenating these two instead of doing it in one function call? For reusability or being able to use existing translation?

Reply:
> being able to use existing translation?

That's the point I thought. 😊

Reply:
Let's take this opportunity to fix up the whole thing. 🙂 

```suggestion
			'description' => __( 'Display the author on the Course Archive and My Courses pages. This setting does not apply when these pages use blocks.', 'sensei-lms' ),
```

Does this describe it well enough?

Reply:
@donnapep 's linguistic skills 💯 ^ 💯